components/header/header.tsx

```diff
@@ -18,17 +16,19 @@ import {
 import headerimage from '../../public/assets/BAQJS.png';
 
 const classes = {
-  item: cn('ml-4 my-1 text-primary-500 text-xl'),
+  item: cn('ml-4 my-1 text-xl'),
 };
 
 export const Header = () => {
   return (
-    <header className={`flex justify-center ${TopBar({ isDark: false })}`}>
+    <header className={`flex justify-center`}>
       <nav className={`container flex flex-col justify-center md:flex-row md:justify-between items-center py-2`}>
-        <Link href="/" className="flex items-center">
-          <Image className="rounded-md" src={headerimage} alt="BarranquillaJS logo" width={30} height={30} />
-          <span className="ml-2">Barranquilla JS</span>
-        </Link>
+        <h1>
```

Comment:
This could be negative for SEO

Reply:
👍🏽

Reply:
I was réfering of the change of all header tags in the tag, make sure all tags are ordered as lighthouse good practices page suggests